kolibri/core/tasks/management/commands/base.py

```diff
@@ -22,7 +23,17 @@ def __init__(self, total=100, level=0, update_callback=None):
         self.level = level
         self.update_callback = update_callback
 
+        try:
+            # Check we are executing inside a click context
+            # as we only want to display progress bars from the command line
+            click.get_current_context()
+            self.progressbar = click.progressbar(length=total, width=0)
+        except RuntimeError:
+            self.progressbar = None
+
     def update_progress(self, increment=1, message="", extra_data=None):
+        if self.progressbar:
+            self.progressbar.update(increment)
```

Comment:
neat!
![image](https://user-images.githubusercontent.com/2367265/67343467-35951c80-f4ea-11e9-8637-422dde22b7a5.png)


doesn't do a great job of handling terminal resizes in mac, but that's not a deal-breaker:

![image](https://user-images.githubusercontent.com/2367265/67343495-4b0a4680-f4ea-11e9-80fa-8b9499ebe96b.png)

Reply:
Yeah - I couldn't work out whether that was my fault or Click's.